pyfolio/plotting.py

```diff
@@ -30,18 +30,21 @@
 from . import pos
 
 
-def set_plot_defaults():
-    """
-    Sets default plotting/charting colors/styles.
-    """
+def set_context(context='notebook', font_scale=1.5, rc=None):
```

Comment:
IMO `set_context` is a misleading name for something that's still being used as a context manager.  I would just call this `context` and use it as:

```
with plotting.context(...):
    do stuff
```